.openpublishing.redirection.json

```
@@ -551,6 +551,10 @@
         {
             "source_path": "docs/docs-ref-conceptual/requirement-sets/word-api-requirement-sets.md",
             "redirect_url": "/javascript/api/requirement-sets/word/word-api-requirement-sets"
+        },
+        {
+            "source_path": "/javascript/api/excel/excel.documenttaskchangeproperties",
```

Actually, I think a redirection isn't needed because this object was in preview.

However, if you choose to add it, then the source path should look like my suggestion. For future, note that if it applies to multiple requirement sets, then you'd need to add an entry for each affected version.

```suggestion
            "source_path": "docs/docs-ref-autogen/excel/excel/excel.documenttaskchangeproperties.yml",
```